package.json

```diff
@@ -15,7 +15,7 @@
     "test": "npm run unit-test",
     "posttest": "aud --production",
     "type-check": "tsc",
-    "unit-test": "istanbul cover node_modules/mocha/bin/_mocha tests/lib/**/*.js tests/util/**/*.js tests/index.js",
+    "unit-test": "istanbul cover node_modules/mocha/bin/_mocha tests/lib/rules/no-render-return-undefined.js",
```

Comment:
commenting here to ensure this doesn't get merged. fwiw you can `npx mocha tests/lib/rules/no-render-return-undefined.js` to run just that one file locally :-)